templates/step/edit.html.ep

```diff
@@ -15,6 +15,14 @@
     % }
     setup_needle_editor( <%= b(encode_json($default_needle)) %> );
     setupResultButtons();
+    % for my $pro (@$properties) {
+        % if ($pro->{name} eq "workaround") {
+            $("#property_workaround").prop("checked", true);
+            $("#input_workaround_desc").val("<%= $pro->{value} %>");
```

Comment:
Using Mojo's HTML escaping is not suited for making a JavaScript-string, right @kraih ?. We usually add strings needed by JavaScript code as data attribute of some related HTML element.

Reply:
I will remove the code. The function setup_needle_editor in javascript has done this.

Reply:
Right, @Martchus.